metadata.rb

```diff
@@ -5,11 +5,11 @@
 description       'Application cookbook for installing and configuring Vault.'
 issues_url        'https://github.com/sous-chefs/vault/issues'
 source_url        'https://github.com/sous-chefs/vault'
-chef_version      '>= 13'
+chef_version      '>= 15'
 version           '4.3.0'
 
 supports 'ubuntu'
 supports 'redhat'
 supports 'centos'
 
-depends 'ark', '~> 4.0.0'
+depends 'ark', '~> 5.0.0'
```

Comment:
Does this still need to be a pessimistic lock? I think we prefer not to do that if possible.

Reply:
What would be the consensus going forward? `ark` is one of ours now so it should be stable within versions so I'd think it'd OK to leave it pessimistic.

Reply:
Since we own it now it should be OK to keep this.